manifests/kustomize/base/argo/workflow-controller-deployment.yaml

```diff
@@ -27,7 +27,8 @@ spec:
         - --configmap
         - workflow-controller-configmap
         - --executor-image
-        - gcr.io/ml-pipeline/argoexec:v2.7.5-license-compliance
+        - gcr.io/ml-pipeline/argoexec:v2.11.6-license-compliance
+        - --namespaced
```

Comment:
This is the command that I need to specifically add for my deployment.

Reply:
Is there a corresponding documentation? Does omitting the argument make it cluster mode? We need to make sure it is customizable from an overlay to continue support multi-user mode.

Reply:
The cluster vs namespace install seems to start since [v2.5](https://argoproj.github.io/argo/managed-namespace/) - this can also be found in the [v2.5 release branch](https://github.com/argoproj/argo/blob/release-2.5/manifests/namespace-install/overlays/workflow-controller-deployment.yaml). 

In my case, it did become a cluster install if I simply replace the image version to ` argoproj/argoexec:v2.11.1` - I got permission issues when trying to launch new pipelines.

I am not familiar with the `multi-user` mode. Does it install the workflow-controller in cluster scope?

Reply:
Error log from when trying to launch pipelines without `--namespaced` 
```bash
E1107 01:14:06.943364       1 reflector.go:153] pkg/mod/k8s.io/client-go@v0.17.8/tools/cache/reflector.go:105: Failed to list *unstructured.Unstructured: workflowtemplates.argoproj.io is forbidden: User "system:serviceaccount:kubeflow:argo" cannot list resource "workflowtemplates" in API group "argoproj.io" at the cluster scope
```